### PR TITLE
feat: Add tests for `mesh_utils`

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
## Overview
Add test for `Mesh` class in `mesh_utils.py` file. I can't find a way to verify that the functions are making the expected changes (except for the write and read functions), so I only assure that this class runs with no issues. 

Also, I can't fully cover internal functions of the class, as the public functions are not using the whole code of the functions. We might need to prune these functions, as some of their branches will never be used with the current code.

### CODE REVIEW

File: src/ansys/meshing/prime/graphics/graphics.py [hunk 1]
```diff
@@ -679,7 +680,7 @@ def get_color_by_type(self) -> ColorByType:
         return self._color_by_type
 
 
-class _DisplayMesh(object):
+class _DisplayMesh(object):  # pragma: no cover
     """ """
```
Comment (RobPasMue) [inline src/ansys/meshing/prime/graphics/graphics.py:684]: We are not using this class?

File: src/ansys/meshing/prime/graphics/graphics.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 from ansys.meshing.prime.internals import defaults
 
 
+# TODO: nodes parameter is unused.
 def compute_face_list_from_structured_nodes(nodes, dim):
     flist = []
```
Comment (RobPasMue) [inline src/ansys/meshing/prime/graphics/graphics.py:16]: Then remove it... but it's very weird that it is not being used.

File: src/ansys/meshing/prime/graphics/graphics.py [hunk 3]
```diff
@@ -404,19 +405,19 @@ def __hide_unhide_selection(self, flag):
                 for disp_mesh in data["faces"]
             ]
 
-    def __show_edges_callback(self, flag):
+    def __show_edges_callback(self, flag):  # pragma: no cover
         [
```
Comment (RobPasMue) [inline src/ansys/meshing/prime/graphics/graphics.py:409]: We shouldn't over use ``# pragma: no cover`` if possible... is there no way to test these internal functions? If they are unreachable we should consider removing them.

File: tests/lucid/test_mesh_utils.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+
+def test_wrap(mesh):
+    mesh.wrap()
+    mesh.wrap(contact_prevention_size=0.01)
+
+
+def test_delete_topo(mesh):
+    mesh.delete_topology()
+    mesh.delete_topology(delete_edges=True)
```
Comment (RobPasMue) [inline tests/lucid/test_mesh_utils.py:81]: Can we check some kind of internal values to see if the tests have run successfully? I don't like having tests that only check whether a function call is successfull or not. Let's check their outputs if any at least.

File: src/ansys/meshing/prime/graphics/graphics.py (hunk 2 above)
Comment (AlejandroFernandezLuces) [reply src/ansys/meshing/prime/graphics/graphics.py:16]: I wanted to ask @ninad-kamat if this was intended or not, there might be something missing in this function that needed the `nodes` parameter. We can remove it if this is just a mistake.

**Review by RobPasMue — APPROVED** [review]
After opening #489 - LGTM

**Pull request merged**